lib/pom.xml

```diff
@@ -141,6 +142,77 @@
                     <target>${version.jdk}</target>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>buildnumber-maven-plugin</artifactId>
+                <version>1.4</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>create</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <getRevisionOnlyOnce>true</getRevisionOnlyOnce>
+                    <shortRevisionLength>7</shortRevisionLength>
+                    <timestampFormat>{0,date,yyyyMMdd}</timestampFormat>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>1.10</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>parse-version</goal>
+                            <goal>regex-property</goal>
+                        </goals>
+                        <configuration>
+                            <name>rpm.release_extra</name>
+                            <value>${parsedVersion.qualifier}git${buildNumber}</value>
+                            <regex>SNAPSHOT</regex>
+                            <replacement>\${timestamp}</replacement>
+                            <failIfNoMatch>false</failIfNoMatch>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>rpm-maven-plugin</artifactId>
```

Comment:
I don't know if you are using travis or not, but I found that the tool does not take kindly to rpm generation. You may need to wrap this in a profile.